src/main/views/macro/support.njk

```diff
@@ -44,9 +44,23 @@
 {% endmacro %}
 
 {% macro _holidayWidget() %}
-  {% if (toDate() < toDate("2021-01-02") and toDate() >= toDate("2020-12-01")) %}
-    <h2 class="heading-medium contact-us"> {{ t('Christmas hours')}}</h2>
-    <p>{{ t("We're closed from 24 to 28 December and 1 January.")}}</p>
+  {% if (toDate() < toDate("2022-04-19") and toDate() >= toDate("2022-04-08")) %}
```

Comment:
to date should be less than or equal to 19th April. As the dates should be visible on 19th april also

Reply:
best thing could be toDate() < toDate("2022-04-20")